webpack/app/scss/_basic.scss

```diff
@@ -125,24 +125,45 @@ footer {
   #portalredes-twitter {
     @include sprite($rede-twitter);
   }
+  #portalredes-twitter.inverter {
+    @include sprite($twitter-branco);
```

Comment:
Ao inves de criar 2 icones, um preto e outro branco, podemos adicionar somente o icone preto, e aplicar o css ``filter: invert(100%);`` para inverter a cor para branco do ícone.

Isso faz com que o arquivo de sprite seja muito menor e possa ser carregado mais rapidamente.

Reply:
Apenas segui o padrão do que já estava desenvolvido, onde alguns ícones tem duas ou mais versões.

Reply:
Mas voce não concorda que fica melhor se for possível ter uma versão do ícone do sprite somente? Qual a dificuldade de alterar (ou remover) o icone e uma linha de CSS?

Reply:
Quanto as versões dos ícones, acredito que existam até alternativar melhores que PNGs de baixa resolução para o site que compromete a visualização do mesmo em telas de alta qualidade ou alguns aparelhos mobile mas, acredito que essa discussão não cabe a este espaço.

Não existe dificuldade em alterar/remover o ícone em uma linha de CSS, aliás, é isto que tem sido feito, neste caso pra mim é indiferente, como disse, só segui o que já existia no sistema, posso usar o invert do CSS e remover as contra-partes em alto contraste do sistema mas, em questão de código, o ganho é pouco, mas, sim, seria menos meia dúzia de ícones a serem carregados. Vou providenciar essas alterações.